app/screens/settings-screen/settings-screen.tsx

```diff
@@ -1,25 +1,36 @@
-import { gql, useApolloClient, useLazyQuery, useQuery } from "@apollo/client"
 import * as React from "react"
 import { Alert, View } from "react-native"
+import Share from "react-native-share"
+import EStyleSheet from "react-native-extended-stylesheet"
 import { Divider, Icon, ListItem } from "react-native-elements"
-import EStyleSheet from 'react-native-extended-stylesheet'
-import Share from 'react-native-share'
+import { gql, useApolloClient, useLazyQuery, useQuery } from "@apollo/client"
+
+// Components
 import { Screen } from "../../components/screen"
 import { VersionComponent } from "../../components/version"
-import { walletIsActive } from "../../graphql/query"
-import { translate } from "../../i18n"
+
+// Constants
+import { language_mapping } from "./language-screen"
 import { palette } from "../../theme/palette"
+import { WHATSAPP_CONTACT_NUMBER, WHATSAPP_DEFAULT_CONTACT_MESSAGE } from "../../constants/support"
+
+// Functions
+import { translate } from "../../i18n"
+import { walletIsActive } from "../../graphql/query"
+
+// Utils
+import { openWhatsApp } from "../../utils/external"
 import { resetDataStore } from "../../utils/logout"
 import { hasFullPermissions, requestPermission } from "../../utils/notifications"
-import { language_mapping } from "./language-screen"
+
 
 const styles = EStyleSheet.create({
   screenStyle: {
     backgroundColor: palette.lighterGrey
   },
 })
 
-export const SettingsScreen = ({navigation}) => {
+export const SettingsScreen: any = ({ navigation }) => {
```

Comment:
we're trying to move forward a strongly type database. could you add the correct type here?

Reply:
Sure.